test/test-coordinator.js

```diff
@@ -316,4 +316,9 @@ TestCoordinator.prototype.getSUTHostPort = function getSUTHostPort() {
     return this.sutHostPort;
 };
 
+TestCoordinator.prototype.checksum = function checksum() {
+    //default to no-checksum. This method is changed based on runtime data of the SUT
```

Comment:
Can you add the description of the pull request to the line above this function signature? In that case, we'll know very well what this method is used for.